extensions/_oci.md

```diff
@@ -32,21 +38,25 @@ Content-Type: application/json
 {
     "extensions: [
         {
-            "name": "_<ns>/<ext>/<component>"
+            "name": "_<ns>/<ext>/<operation>"
         },
     ]    
 }
 ```
 
 ### *Extensions* Property Descriptions
 
-- **`extensions`** *array of extension objects with properties like `name`*
+- **`extensions`** *array of extension objects*, REQUIRED
+
+  This property contains a list of supported extension endpoints.
+
+  The [relative URI]((https://tools.ietf.org/doc/html/rfc1808))
+  which MAY be constructed by replacing
+  the base discovery extension path,`_ext`.
```

Comment:
While this extension is mounted under `_ext`, I don't think URI is relative `_ext`. Or maybe I'm missing something?